### PR TITLE
React Capstone: Crypto Currency Application

### PR DESCRIPTION
**Pull request by dselasea**
In this pull request I -

- Created a single-page application (SPA) built with React and Redux.
- Retrieved from the API should be stored in the Redux store.
- Filtered data from API using a Filter stateless component.
- Created integration tests for your application using the React Testing Library.
- Added a live link to README
- Recorded Video description of the Project
- Added unique routes to links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,139 @@
...
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for creating this project.
```
Comment (prolajumokeoni) [inline README.md]: - [ ] In your acknowledgement section, kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 👍🏿

File: README.md [hunk 2]
```diff
@@ -1,70 +1,139 @@
...
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (prolajumokeoni) [inline README.md:137]: - [ ] When I click on your MIt license link in your readme it is broken, please fix it 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @dselasea 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Created `Home` page and  `Detail` page 👏🏿 
✔️ App is deployed 👏🏿   
✔️  Crypto can be filtered by name 💪🏿 




## Required Changes ♻️

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
99235e31579789 56617c411f336  | Screenshot_822

- [ ] Kindly make your mobile design look more like the Behance layout by adding the alternating card color  layout 🏁  and other card layout features like aligning your image and text, implementing these features will make your design look very close to the Behance design, please use the Typography stated in the Behance design, please don't forget you can use any color you want 👍🏿  


- [ ] Kindly make sure your tests covers several components of the app and some pure functions in Redux code, currently, some tests written are failing please check it 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @dselasea 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**